packages/frontend/src/components/common/Page/Content.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Section } from '../../../providers/TrackingProvider';
+import { SectionName } from '../../../types/Events';
+
+const StyledDiv = styled('div')`
+    padding: 10px 10px;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: stretch;
+    min-width: 0;
+`;
+
+const Content: FC = ({ children }) => (
+    <StyledDiv>
+        <Section name={SectionName.PAGE_CONTENT}>{children}</Section>
```

Comment:
Is this overridden by `<Section>` components in the `children` ? Otherwise the event context is not going to be very useful since they'll all have `section: page_content`

Reply:
Nested sections are supported but I wouldn't describe it as override. The events will use the closest parent section name.

Reply:
Makes sense